gtags-mode.el

```diff
@@ -104,7 +104,7 @@ The address is absolute for remote hosts.")
   "Regex to filter the output with `gtags-mode--output-format-options'.")
 
 (defconst gtags-mode--output-format-options
-  '("--result=cscope" "--path-style=through" "--color=never")
+  '("--result=cscope" "--color=never")
```

Comment:
Please keep the `--path-style` and set a value explicitly

Reply:
Done